components/icu4x/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "icu4x"
+description = "International Components for Unicode"
+version = "0.0.1"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
```

Comment:
Curious, where did 2018 come from?

Reply:
Oh boy! Enjoy - https://doc.rust-lang.org/edition-guide/editions/index.html :)

Reply:
Ah, so it's a reference to major Rust edition and has nothing to do with licence/start of the project. Good to know.